docker-compose.dev-windows.yaml

```diff
@@ -0,0 +1,65 @@
+
+version: '3.8'
+
+networks:
+  app_network:
+    driver: bridge
+
+services:
+  backend:
+    build: .
```

Comment:
you should add path to correct Dockerfile, by default is `Dockerfile`, but you need another one

Reply:
yea, thats needed